.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,29 +1,28 @@
-<!-- ^^^^^
-Please provide a concise, informative and self-explanatory title.
-Don't put issue numbers in there, do this in the PR body below.
-For example, instead of "Fixes #1234" use "Introduce new method to calculate 1+1"
--->
-### 📚 Description
+<!-- Please provide a concise, informative and self-explanatory title. -->
+<!-- Don't put issue numbers in the title. Put it in the Description below. -->
+
+### :books: Description
 
-<!-- Describe your changes here in detail -->
+<!-- Describe your changes here in detail. -->
 <!-- Why is this change required? What problem does it solve? -->
-<!-- If it resolves an open issue, please link to the issue here. For example "Closes #1337" -->
+<!-- If this PR resolves an open issue, please link to it here. For example "Fixes #12345". -->
+<!-- If your change requires a documentation PR, please link it appropriately. -->
 
-### 📝 Checklist
+### :memo: Checklist
 
-<!-- Put an `x` in all the boxes that apply. -->
-<!-- If your change requires a documentation PR, please link it appropriately -->
-<!-- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
+<!-- Put an `x` in all the boxes that apply. It should be `[x]` not `[x ]`. -->
 
-- [ ] I have made sure that the title is self-explanatory and the description concisely explains the PR.
-- [ ] I have linked an issue or discussion.
+- [ ] The title is concise, informative, and self-explanatory.
+- [ ] The description explains in detail what this PR is about.
+- [ ] I have linked a relevant issue or discussion.
 - [ ] I have created tests covering the changes.
 - [ ] I have updated the documentation accordingly.
 
-### ⌛ Dependencies
-<!-- List all open pull requests that this PR logically depends on -->
-<!--
-- #xyz: short description why this is a dependency
-- #abc: ...
+### :hourglass: Dependencies
+
+<!-- List all open PRs that this PR logically depends on
+Depends on #12345: short description why this is a dependency
```

Comment:
Being a list has the advantage that github automatically expands it, and shows the status (merged or not). See https://github.com/sagemath/sage/pull/35114 for an example. This is not the case of the "Depends on" format. I would leave this for now as a list, until we indeed formalize this using a github action.

Reply:
Okay. Done